openfl-tutorials/interactive_api/Tensorflow_MNIST/workspace/Tensorflow_MNIST.ipynb

```diff
@@ -407,6 +407,18 @@
     "fl_experiment = FLExperiment(federation=federation, experiment_name=experiment_name)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "480700cc-400b-4073-9daf-71b1f0d60d62",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# print the default federated learning plan\n",
+    "import openfl.native as fx\n",
+    "print(fx.get_plan(plan_config_path='plan/plan.yaml'))"
```

Comment:
We mix Interactive API with 'native' API here, which is confusing even for me

Reply:
Hi @igor-davidyuk, the idea behind it is that users should have the flexibility to make any changes to the plan before the experiment starts. Right now, only a few parameters can be modified while starting the experiment. Many other fields, like compression pipeline, can't be modified by the users directly through the jupyter notebook. This PR initializes the plan when the FLExperiment object is created and then users can display the plan settings to see the default values. Accordingly, changes could be made to the plan config using the override_config option.

Reply:
I understand the intention, I just say we should not mix different APIs. 
We actually did not address this comment